src/Symfony/Component/Form/Form.php

```diff
@@ -766,10 +766,28 @@ public function getClickedButton()
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getAllErrors()
+    {
+        return $this->getErrorIterator(true, true)->all();
```

Comment:
Why not to do it easier? 
`return iterator_to_array($this->getErrorIterator(true), false);`